build-config/resources/Ballerina.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+org = "ballerina"
+name = "ldap"
+version = "@toml.version@"
+authors = ["Ballerina"]
+export=["ldap"]
+keywords = ["ldap"]
+repository = "https://github.com/ballerina-platform/module-ballerina-ldap"
+license = ["Apache-2.0"]
+distribution = "2201.8.6"
+
+[platform.java17]
+graalvmCompatible = true
+
+[[platform.java17.dependency]]
+groupId = "io.ballerina.lib"
+artifactId = "ldap-native"
+version = "0.1.0"
+path = "../native/build/libs/ldap-native-0.1.0-SNAPSHOT.jar"
```

Comment:
```suggestion
path = "../native/build/libs/ldap-native-@project.version@.jar"
```

Reply:
Fixed in [31c3a85](https://github.com/ballerina-platform/module-ballerina-ldap/pull/45/commits/31c3a85552371b3d20a942de6e0a653f601d91cd)